src/vsphere_cpi/spec/integration/set_vm_metadata_spec.rb

```diff
@@ -4,34 +4,66 @@
 require 'yaml'
 
 describe 'set_vm_metadata' do
-  context 'when called with duplicate keys with multiple threads' do
-    subject(:cpi) { VSphereCloud::Cloud.new(cpi_options) }
-    let(:metadata) { { "bosh-#{SecureRandom.uuid}-key" => "test value" } }
-    let(:network_spec) do
-      {
+  let(:metadata) { {
+      "bosh-#{SecureRandom.uuid}-key" => "test value",
+      "OS" => "Linux",
+      "BU" => "MAPBU",
+      "Team" => "kaitingc", #non-existent category
+      "Type" => "Sandbox",#tag does not match
+      "deployment" => "zookeeper"
```

Comment:
Remove personal email alias like `kaitingc` from here. Let's change it to `vcpi-team`